manifests/state-nv-ipam-cni/040-nv-ipam-controller.yaml

```diff
@@ -0,0 +1,112 @@
+# 2023 NVIDIA CORPORATION & AFFILIATES
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: nv-ipam-controller
+  namespace: {{ .RuntimeSpec.Namespace }}
+  annotations:
+    kubernetes.io/description: |
+      This deployment launches the nv-ipam controller for nv-ipam.
+  labels:
+    component: nv-ipam-controller
+    app: nv-ipam
+    name: nv-ipam-controller
+spec:
+  strategy:
+    type: RollingUpdate
+  replicas: 1
+  selector:
+    matchLabels:
+      name: nv-ipam-controller
+  template:
+    metadata:
+      labels:
+        component: nv-ipam-controller
+        app: nv-ipam
+        name: nv-ipam-controller
+    spec:
+      priorityClassName: system-cluster-critical
+      serviceAccountName: nv-ipam-controller
+      # required to be scheduled on a linux node with node-role.kubernetes.io/master label and
```

Comment:
Please, add `"beta.kubernetes.io/os": "linux"` selector or delete this comment

Reply:
will delete comment